tests/SprykerSdkTest/Zed/AppSdk/Business/AppManifestValidatorFacadeTest.php

```diff
@@ -100,16 +100,14 @@ public function testValidateManifestReturnsFailedResponseWhenPageBlockTypeFieldH
      */
     public function testValidateManifestReturnsFailedResponseWhenFilesNotFound(): void
     {
-        // Act
+       // Act
         $validateResponseTransfer = $this->tester->getFacade()->validateManifest(
             $this->tester->haveValidateRequest(),
         );
 
-        // Assert
-        $this->assertCount(1, $validateResponseTransfer->getErrors());
-
+    // Assert
         $expectedErrorMessage = $validateResponseTransfer->getErrors()[0];
-        $this->assertSame('No manifest files found.', $expectedErrorMessage->getMessage());
+        $this->assertEquals('No manifest files found.', $expectedErrorMessage->getMessage(), 'Manifest file "vfs://root/config/app/manifest/en_US.json" not found');
```

Comment:
```suggestion
        $this->assertSame('No manifest files found.', $expectedErrorMessage->getMessage(), 'Manifest file "vfs://root/config/app/manifest/en_US.json" not found');
```

Try to only use `assertSame` it's the better choice `assertEquals` can return true where it shouldn't.